backend/fms_core/models/sample_property.py

```diff
@@ -0,0 +1,19 @@
+import reversion
+
+from .tracked_model import TrackedModel
+from django.db import models
+from .sample import Sample
+
+from ._constants import STANDARD_NAME_FIELD_LENGTH
+from ._validators import name_validator
+
+
+@reversion.register()
+class SampleProperty(TrackedModel):
```

Comment:
I would not call this property even if this is close in many way to the properties ... We will just get confused and it will also be hard to explain to the users why some properties can be accessed in a way in some other another way. We often call them additional meta-data with David, but this is a bit long ... We could invent a term for them Sample Addtributes ? MetAdData ? eheh well maybe it's time for a brainstorming meeting ...